app/src/main/java/tech/okcredit/oklayoutinflator/MainActivity.kt

```diff
@@ -32,10 +26,4 @@ class MainActivity : AppCompatActivity() {
             .replace(binding.mainContent.container.id, MainFragment.newInstance())
             .commitNow()
     }
-
-    override fun onDestroy() {
-        super.onDestroy()
-        _binding = null
```

Comment:
Can we revert changes over binding and keep `_binding = null`. 

When ViewBinding is not cleared or set to null, it will not be eligible for GC.

Reply:
Setting the ViewBinding to `null` is only required in the case of a `Fragment` because it may outlive the view's lifecycle.
In the case of an `Activity`, the field will be automatically GCed.
Check this: https://developer.android.com/topic/libraries/view-binding

Reply:
Agreed